cmd/manager/main.go

```diff
@@ -143,12 +143,12 @@ func main() {
 		stopSignalContext.Done(),
 	)
 	if err != nil {
-		klog.Fatalf("failed to create feature gate accessor: %w", err)
+		klog.Fatalf("failed to create feature gate accessor: %v", err)
 	}
 
 	featureGates, err := awaitEnabledFeatureGates(featureGateAccessor, 1*time.Minute)
 	if err != nil {
-		klog.Fatalf("failed to get feature gates: %w", err)
+		klog.Fatalf("failed to get feature gates: %v", err)
```

Comment:
Errors should always be `%w` in format strings now

Reply:
Arguments to `klog.Fatalf ` are handled same as `fmt.Printf`, which doesn't support `%w` and `go vet` is reporting error for the same. 
```
cmd/manager/main.go:146:3: k8s.io/klog/v2.Fatalf does not support error-wrapping directive %w
cmd/manager/main.go:151:3: k8s.io/klog/v2.Fatalf does not support error-wrapping directive %w
```
Yeah, instead of `%v` we can use `%s`. I will make the change.

Reply:
Ah I did not realise this, my bad!